tests/runtime/other

```diff
@@ -68,6 +68,11 @@ RUN bpftrace -e 'BEGIN { printf("-%s-\n", str($1)); exit() }'
 EXPECT --
 TIMEOUT 1
 
+NAME positional arg count
+RUN bpftrace -v -e 'BEGIN { if ($# < 2) { printf("expected 2 args\n");} exit();}' "one"
+EXPECT expected 2 args
+TIMEOUT 5
+
```

Comment:
Could you write another runtime tests checking the happy case with something like:

```
RUN bpftrace -v -e 'BEGIN { printf("got %d args: %s %d\n", $#, str($1), $2);} exit();}' "omg" 314
EXPECT got 2 args: omg 314
```

Reply:
I'll just switch my test for this — I don't think mine adds more.